fieldline_client/lib.py

```diff
@@ -242,7 +239,7 @@ def init_acquisition():
 def parse_data(data):
     global channel_key_list
     global data_stream_multiplier
-    chunk = numpy.zeros((len(data),num_working_sensors()), dtype=numpy.single)
+    chunk = np.zeros((len(data),num_working_sensors()), dtype=np.single)
```

Comment:
`np` is the convention for `numpy` that is universally accepted

Reply:
really? Coming new to python accept my comments as those surging from curiosity. But really? If the name of the thing is numpy, who thinks is a good idea to "save" >>three<< letters?

Reply:
lol :) I would think of it in the same way as Journal abbreviations. Nothing wrong in using the full name, but if you use the abbreviation, use `np`. I just changed it to `np` because that's what I've seen more.

Reply:
> I just changed it to np because that's what I've seen more.
please continue. Just reminded me of this rant I came through some years ago... 

http://lkml.iu.edu/hypermail/linux/kernel/1510.3/02866.html

Reply:
:) Yeah I've seen that one. I'm actually done here. Setting the PR to [MRG].